README.md

```diff
@@ -1,4 +1,67 @@
 # sync-music
+[![Build Status](https://travis-ci.org/fossdelhi/sync-music.svg?branch=syncing)](https://travis-ci.org/fossdelhi/sync-music)
+
 A project, to ease backup, add metadata, and sync your music using cloud storage. :)
 
-Disclaimer: A repo on an organisation is still signed by your *personal* gpg key
+This app is build up with Python [3.5, 3.6] and Shell [Bash: 4.3] Script.
```

Comment:
There can be only one version of Python which you state as build. It's better to state `Tested on` instead.